data/hai.txt

```diff
@@ -49,6 +49,46 @@ fleet "Small Hai"
 	variant
 		"Lightning Bug"
 		"Lightning Bug (Pulse)" 2
+	variant 2
+		"Centipede"
+	variant 3
+		"Aphid" 3
+		"Centipede"
+	variant
+		"Centipede" 3
+	variant
+		"Centipede" 2
+		"Lightning Bug" 2
+	variant 2
+		"Centipede" 2
+		"Water Bug"
+	variant
+		"Centipede"
+		"Lightning Bug" 2
+	variant
+		"Centipede"
+		"Aphid" 5
+	variant
+		"Centipede" 2
+		"Lightning Bug" 2
+	variant
+		"Centipede" 2
+		"Lightning Bug" 
+		variant
+		"Grasshopper"
+	variant
+		"Grasshopper"
+		"Centipede" 2
+	variant
+		"Grasshopper" 2
+		"Centipede" 
+	variant
+		"Lightning Bug" 2
+		"Grasshopper" 2
+	variant 1
+		"Geocoris"
```

Comment:
There probably shouldn't be any small fleets able to spawn the Geocoris, given that it's similar in size to the Shield Beetle.